tv_apps/smart-system/js/crash_reporter.js

```diff
@@ -26,9 +25,20 @@ var CrashReporter = (function() {
 
   // This function should only ever be called once.
   function showDialog(crashID, isChrome) {
-    var title = isChrome ? _('crash-dialog-os2') :
-      _('crash-dialog-app', { name: crashedAppName });
-    document.getElementById('crash-dialog-title').textContent = title;
+    if(isChrome) {
+      var title = {
+        id: 'crash-dialog-os2',
+        args: null
+      };
+    } else {
+      var title = {
+        id: 'crash-dialog-app',
+        args: { name: crashedAppName };
+      };
+    }
+
+    document.getElementById('crash-dialog-title').setAttribute('data-l10n-id',
```

Comment:
Is this working?

According to the patch [1], we should use `setAttribtues` here:
```
  var elem = document.getElementById('crash-dialog-title');
  if (isChrome) {
     navigator.mozL10n.setAttributes(elem, 'crash-dialog-os2');
   } else {
     navigator.mozL10n.setAttributes(elem,
     'crash-dialog-app', { name: crashedAppName });
  }
```

[1] https://github.com/mozilla-b2g/gaia/commit/52ee3c33aacd17ba32de3e187ecfc554ef127993